packages/console/src/onboarding/pages/Welcome/index.tsx

```diff
@@ -81,20 +79,73 @@ function Welcome() {
                 )}
               />
             </FormField>
+            {/* Check whether it is a business use case */}
+            {watch('project') === Project.Company && (
+              <>
+                <FormField
+                  isMultiple
+                  title="cloud.welcome.title_field"
+                  headlineClassName={styles.cardFieldHeadline}
+                >
+                  <Controller
+                    control={control}
+                    name="titles"
+                    render={({ field: { onChange, value } }) => (
+                      <MultiCardSelector
+                        className={styles.titleSelector}
+                        optionClassName={styles.option}
+                        value={value ?? []}
+                        options={titleOptions}
+                        onChange={(value) => {
+                          onChange(value.length === 0 ? undefined : value);
+                        }}
+                      />
+                    )}
+                  />
+                </FormField>
+                <FormField title="cloud.welcome.company_name_field">
+                  <TextInput
+                    placeholder={t('cloud.welcome.company_name_placeholder')}
+                    {...register('companyName')}
+                  />
+                </FormField>
+                <FormField
+                  title="cloud.welcome.company_size_field"
+                  headlineClassName={styles.cardFieldHeadline}
+                >
+                  <Controller
+                    control={control}
+                    name="companySize"
+                    render={({ field: { onChange, value, name } }) => (
+                      <CardSelector
+                        name={name}
+                        value={value ?? ''}
+                        options={companySizeOptions}
+                        optionClassName={styles.option}
+                        onChange={(value) => {
+                          onChange(conditional(value && value));
```

Comment:
`conditional(value)`

Reply:
`value` here is string type, this implementation avoids the empty string.